messaging/README.md

```diff
@@ -0,0 +1,46 @@
+# Padrões para RabbitMQ na Liber
```

Comment:
Precisamos tomar uma decisão com esse PR. Ou passamos tudo já feito pra português ou traduzimos ele pra inglês 😬

Reply:
I prefer English, but I think that our new pattern to document things is in Portuguese :cry:

Reply:
O padrão das docs é português porque hoje temos vários níveis diferentes de inglês na Liber, então exigir inglês pode inibir a contribuição ou dificultar a comunicação em diversos momentos. 
Dito isso, quando as alterações são adicionadas linha a linha é mais de boa e o review pode pegar erros de inglês, igual já aconteceu em outro PR desse repo, mas como nesse caso é um texto grande eu diria pra gente mergear em português mesmo pra não perder tempo com isso agora. O que vc acha?

Reply:
Por mim parece um bom plano @guilpejon